app/components/Nav/Mobile/index.tsx

```diff
@@ -0,0 +1,118 @@
+  const [showMenu, setShowMenu] = React.useState(false)
+  const [showArticles, setShowArticles] = React.useState(false)
+  const toggleMenu = () => {
+    setShowMenu(!showMenu)
+    setShowArticles(false)
+  }
+  console.log('toc', toc)
+  return (
+    <header className="top-header">
+      {!showArticles && (
+        <>
+          <nav className="top-nav">
+            <Link to="/" className="top-logo">
+              <AISafetyIcon />
+            </Link>
+            {!showMenu ? (
+              <ListLarge className={'menu-button'} onClick={toggleMenu} />
+            ) : (
+              <XLarge className={'menu-button'} onClick={toggleMenu} />
+            )}
+          </nav>
+          {showMenu && (
+            <div className="mobile-menu">
+              <Button
+                action={() => setShowArticles(!showArticles)}
+                className="secondary full-width space-between"
+                icon={<CarrotLarge />}
+              >
+                <p className={'composite-elements'}>
+                  <OpenBookIcon className={'icon-margin'} />
+                  Articles
+                </p>
+              </Button>
+              <Button action="https://xkcd.com/285/" className="secondary full-width flex-start">
+                <p className={'composite-elements'}>
+                  <BotIcon className={'icon-margin'} />
+                  Stampy chatbot
+                </p>
+              </Button>
+            </div>
+          )}
+        </>
+      )}
+      {showArticles && (
+        <>
+          <nav className="articles-header">
+            <CarrotLarge className={'back-icon'} onClick={() => setShowArticles(false)} />
+            <p className={'composite-elements'}>
+              <OpenBookIcon className={'icon-margin'} />
+              Articles
+            </p>
+            <XLarge className={'menu-button'} onClick={toggleMenu} />
+          </nav>
+
+          <div className={'articles-mobile'}>
+            <div className="default-bold margin-bottom">Introductory</div>
+            {toc
+              .filter((item) => item.category === 'Introductory')
+              .map((item: TOCItem) => (
+                <Link
+                  key={`${item.pageid}-${item.title}`}
+                  className="articles-entry"
+                  to={questionUrl(item)}
+                >
+                  {item.title}
+                </Link>
+              ))}
+            <div className="default-bold margin-bottom top-margin">Advanced</div>
+            {toc
+              .filter((item) => item.category === 'Advanced')
+              .map((item: TOCItem) => (
+                <Link
+                  key={`${item.pageid}-${item.title}`}
+                  to={questionUrl(item)}
+                  className="articles-entry"
+                >
+                  {item.title}
+                </Link>
+              ))}
+            <div className="default-bold margin-bottom top-margin">Browse by category</div>
```

Comment:
what are the classes margin-bottom and top-margin? For bottom padding we want to use the class "padding-bottom-[number]". Maybe I will create padding-top classes and even margin classes too...